how-to/how-to-use-kata-containers-with-nemu.md

```diff
@@ -0,0 +1,120 @@
+- Reduces the number of lines of code.
+- Removes all legacy devices.
+- Reduces the emulation as far as possible.
+
+## How to get started
+
+This section describes how to run Kata Containers with NEMU, first by explaining how to download, build and install it. Then this document walks through the steps needed to update your Kata Containers configuration in order to run with NEMU. 
+
+### Pre-requisites
+This document requires Kata Containers to be [installed](https://github.com/kata-containers/documentation/blob/master/install/README.md) on your system.
+
+Also, it's worth noting that NEMU only supports `x86` and `ARM` architecture.
+
+### NEMU
+
+#### Download, build and install
+
+```bash
+$ git clone https://github.com/intel/nemu.git
+$ cd nemu
+$ git fetch origin/master
+$ git checkout experiment/automatic-removal
+```
+__x86__
+```
+$ SRCDIR=$PWD ./tools/build_x86_64_virt.sh
+```
+__ARM__
+```
+$ SRCDIR=$PWD ./tools/build_aarch64.sh
+```
+
+___Note:__ The branch `experiment/automatic-removal` is a branch published by Jenkins after it applied the automatic removal script to the `topic/virt-x86` branch. The purpose of this code removal being to reduce the source tree by removing files not being used by NEMU._
+
+After those commands successfully returned, you will find the NEMU binary at `$HOME/build-x86_64/x86_64_virt-softmmu/qemu-system-x86_64_virt` (__x86__), or `$HOME/build-aarch64/aarch64-softmmu/qemu-system-aarch64` (__ARM__).
+
+You also need the `OVMF` firmware in order to boot the virtual machine's kernel. It can currently be found at this [location](https://github.com/rbradford/edk2/releases/). Please choose the latest version of this firmware (`v0.2` in this example):
+```bash
+$ mkdir -p /usr/share/nemu/firmware
+$ curl -o /usr/share/nemu/firmware/OVMF.fd https://github.com/rbradford/edk2/releases/download/0.2/OVMF.fd
+```
+___Note:__ The OVMF firmware will be located at this temporary location until the changes can be pushed upstream._
+
+
+### Configure Kata Containers
+All you need from this section is to modify the configuration file `configuration.toml`, to specify the options related to the hypervisor.
+___`/usr/share/defaults/kata-containers/configuration.toml`___
+```diff
+ [hypervisor.qemu]
+-path = "/usr/bin/qemu-lite-system-x86_64"
++path = "/home/foo/build-x86_64/x86_64_virt-softmmu/qemu-system-x86_64_virt"
+ kernel = "/usr/share/kata-containers/vmlinuz.container"
+ initrd = "/usr/share/kata-containers/kata-containers-initrd.img"
+ image = "/usr/share/kata-containers/kata-containers.img"
+-machine_type = "pc"
++machine_type = "virt"
+ 
+ # Optional space-separated list of options to pass to the guest kernel.
+ # For example, use `kernel_params = "vsyscall=emulate"` if you are having
+@@ -31,7 +31,7 @@
+ 
+ # Path to the firmware.
+ # If you want that qemu uses the default firmware leave this option empty
+-firmware = ""
++firmware = "/home/foo/nemu_firmware/OVMF.fd"
```

Comment:
taking in consideration the above step, the location should be: `/usr/share/nemu/firmware/OVMF.fd`